config/config.exs

```diff
@@ -21,13 +21,13 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
-# It is also possible to import configuration files, relative to this
-# directory. For example, you can emulate configuration per environment
-# by uncommenting the line below and defining dev.exs, test.exs and such.
-# Configuration from the imported file will override the ones defined
-# here (which is why it is important to import them last).
-#
-import_config "#{Mix.env}.exs"
-
+if Mix.env == :test do
```

Comment:
If I understand correctly the issue, would it be `Mix.env == :test || Mix.env == :dev`?
And if so, would duplicating the `test.exs` file as `dev.exs` be cleaner?

Reply:
With the change in the PR this works with `mix test.watch`. We also need to consider other environments that are not controlled by us, i.e. `MIX_ENV=staging`.